pbsmetrics/go_metrics.go

```diff
@@ -48,9 +48,13 @@ type Metrics struct {
 	ImpsTypeAudio  metrics.Meter
 	ImpsTypeNative metrics.Meter
 
+	// Notification timeout metrics
 	TimeoutNotificationSuccess metrics.Meter
 	TimeoutNotificationFailure metrics.Meter
 
+	// TCF adaption metrics
+	TCFReqVersion []metrics.Meter
```

Comment:
I'd agree with Scott here about having indexes of a slice correlate with the TCF versions can be a little fragile. We should probably create a map here with the keys being of type `TCFVersionValue`